Tests/Consumer/SwiftMailerConsumerTest.php

```diff
@@ -72,6 +80,8 @@ public function testSendEmail()
         $mail->expects($this->once())->method('setFrom')->with($this->equalTo(array('from@mail.fr' => 'nameFrom')))->willReturnSelf();
         $mail->expects($this->once())->method('setTo')->with($this->equalTo(array('to1@mail.fr', 'to2@mail.fr' => 'nameTo2')))->willReturnSelf();
         $mail->expects($this->once())->method('setReplyTo')->with($this->equalTo(array('replyTo1@mail.fr', 'replyTo2@mail.fr' => 'nameReplyTo2')))->willReturnSelf();
+        $mail->expects($this->once())->method('setCc')->with($this->equalTo(array('cc1@mail.fr', 'cc2@mail.fr' => 'nameCc2')))->willReturnSelf();
+        $mail->expects($this->once())->method('setBcc')->with($this->equalTo(array('bcc1@mail.fr', 'bcc2@mail.fr' => 'nameBcc2')))->willReturnSelf();
```

Comment:
Well, refactoring this class isn't the point of this PR. I'll stick with the original style.